projects/plugins/jetpack/_inc/client/search/dashboard/index.jsx

```diff
@@ -34,14 +37,24 @@ const useComponentWillMount = func => {
  */
 function SearchDashboard( props ) {
 	// NOTE: API root and nonce must be set before any components are mounted!
-	const { apiRootUrl, apiNonce, setInitialState: setSearchDashboardInitialState } = props;
+	const {
+		apiRootUrl,
+		apiNonce,
+		setInitialState: setSearchDashboardInitialState,
+		siteConnectionStatus,
+		siteAdminUrl,
+	} = props;
 
 	useComponentWillMount( () => {
 		apiRootUrl && restApi.setApiRoot( apiRootUrl );
 		apiNonce && restApi.setApiNonce( apiNonce );
 		setSearchDashboardInitialState && setSearchDashboardInitialState();
 	} );
 
+	const aboutPageUrl = siteConnectionStatus
+		? siteAdminUrl + 'admin.php?page=jetpack_about'
+		: getRedirectUrl( 'jetpack' );
```

Comment:
I'm assuming this changes the logo href depending on the connection status? If so, are users able to navigate to this page with an unconnected Jetpack site?

Reply:
You're right. User should not be able to access this page if not connected. Although it will make sense when Search becomes a separate plugin, still over thinking it is. 

Changed in d2341a0da